src/components/Menu/Menu.style.ts

```diff
@@ -70,3 +71,36 @@ export const buttonSpanStyle = ({
     marginRight: hasChildren ? theme.spacing.sm : 0,
   },
 });
+
+type MenuOptions = {
+  menuPosition?: 'left' | 'right';
```

Comment:
add a type in the menu.tsx that would be like the below and then use it everywhere you want

```
type MenuPositionAllowed = 'left' | 'right';

type Props = {
  ...
  menuPosition?: MenuPositionAllowed;
  ...
}
```

Reply:
this must be imported from the menu.tsx